docs/src/content/docs/architecture/milestones.md

```
@@ -0,0 +1,88 @@
+---
+title: Approved Milestones
+description: The structured roadmap and definition of done for successive versions of the Dice Chess Engine. See AGENTS.md for the canonical project state.
+---
+
+Assign tasks to these milestones logically. Each milestone must be fully tested (including performance benchmarks) before moving to the next.
+
+[View current milestones on GitHub](https://github.com/fortemate/dicechess-engine/milestones?sort=title&direction=asc)
+
+---
+
+### ✅ v0.1 - Foundation & Core Types
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)